---a/net/minecraft/block/properties/PropertyBool.java
+++b/net/minecraft/block/properties/PropertyBool.java
@@ -1,35 +1,30 @@
 package net.minecraft.block.properties;
 
-import com.google.common.collect.ImmutableSet;
 import java.util.Collection;
 
-public class PropertyBool extends PropertyHelper
-{
+import com.google.common.collect.ImmutableSet;
+
+public class PropertyBool extends PropertyHelper {
+
     private final ImmutableSet allowedValues = ImmutableSet.of(Boolean.valueOf(true), Boolean.valueOf(false));
     private static final String __OBFID = "CL_00002017";
 
-    protected PropertyBool(String name)
-    {
+    protected PropertyBool(String name) {
         super(name, Boolean.class);
     }
 
-    public Collection getAllowedValues()
-    {
+    public Collection getAllowedValues() {
         return this.allowedValues;
     }
 
-    public static PropertyBool create(String name)
-    {
+    public static PropertyBool create(String name) {
         return new PropertyBool(name);
     }
 
-    public String getName0(Boolean value)
-    {
+    public String getName0(Boolean value) {
         return value.toString();
     }
 
-    public String getName(Comparable value)
-    {
-        return this.getName0((Boolean)value);
+    public String getName(Comparable value) {
+        return this.getName0((Boolean) value);
     }
-}
+}
